src/pages/index.js

```diff
@@ -207,10 +207,10 @@ function Home() {
             <div className="container">
               {<div className="row">
                 <VersionBanner
-                  link="/docs/release-notes/v12-overview"
+                  link="docs/release-notes/v12.3.0"
                   majorVersion="12"
-                  subTitle="Click here and learn about the new features in this new v12."
-                  title="Mia-Platform v12 is now Generally Available"
+                  subTitle="Click here for further information"
+                  title="The new look and feel of Mia-Platform Console is official released!"
```

Comment:
That's nice and I honestly like to give more value to this banner, but that would be the first time we change the banner from a major announcemnet, are we sure about this?


Also you need to change the `majorVersion` attribute value as it will make the banner re-appear to those user who dismissed it

Reply:
Yes, this were agreed togheter with marketing and comunication.

For the major version change i do not get it, why it cannot remain v12?

Reply:
Great, for the majorVersion attribute it's a technical caveat, it is used as identifier to remember whether the user has dismissed the banner or not, leaving it as-is will make the banner visible only to those users who did not dismiss it

Reply:
ahh clear thanks. What version you suggest? 13?

Reply:
since it's a string we could use 12.3 (even though it's not a "major" only)